Px.Utils/Validation/SyntaxValidation/SyntaxValidationFunctions.cs

```diff
@@ -0,0 +1,843 @@
+﻿using PxUtils.PxFile;
+using System.Globalization;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace PxUtils.Validation.SyntaxValidation
+{
+    public delegate ValidationFeedbackItem? ValidationFunctionDelegate(ValidationObject validationObject, PxFileSyntaxConf syntaxConf);
```

Comment:
Maybe split the delegate in three subtypes for each validation object type. This way the validation functions can have more strict parameter types.